src/System.Reflection.Metadata/tests/TestUtilities/LoaderUtilities.cs

```diff
@@ -33,7 +34,18 @@ public static void LoadPEAndValidate(byte[] peImage, Action<PEReader> validator,
                 }
             }
```

Comment:
Is it possible that the file isn't getting immediately unlocked because we still have something using the handle somewhere or something?  I just want to make sure we're not masking an underlying issue.